Packs/CommonPlaybooks/Playbooks/playbook-Search_Endpoints_By_Hash_-_Generic_V2.yml

```diff
@@ -34,34 +34,26 @@ tasks:
     quietmode: 0
   "5":
     id: "5"
-    taskid: c185447e-549c-476e-8477-98c461dadd16
+    taskid: e540cdba-a994-4570-879c-17357fcbd5bf
     type: playbook
     task:
-      id: c185447e-549c-476e-8477-98c461dadd16
+      id: e540cdba-a994-4570-879c-17357fcbd5bf
       version: -1
-      name: Search Endpoints By Hash - CrowdStrike
-      description: Hunt for endpoint activity involving hash and domain IOCs, using Crowdstrike Falcon Host
-      playbookName: Search Endpoints By Hash - CrowdStrike
+      name: CrowdStrike Falcon - Search Endpoints By Hash
+      playbookName: CrowdStrike Falcon - Search Endpoints By Hash
       type: playbook
       iscommand: false
+      description: ''
       brand: ""
     nexttasks:
       '#none#':
       - "12"
     scriptarguments:
-      MD5Hash:
-        complex:
-          root: inputs
-          accessor: MD5Hash
-      SHA1Hash:
+      FileSha256:
         complex:
-          root: inputs
-          accessor: SHA1Hash
-      SHA256Hash:
-        complex:
-          root: inputs
-          accessor: SHA256Hash
+          root: inputs.SHA256Hash
     separatecontext: true
+    continueonerrortype: ""
     loop:
```

Comment:
Is there a loop ?!?! or its something historical?

Reply:
@ssokolovich  I didn't touch it so it must be historical, should i try to remove it?

Reply:
@ssokolovich It seems the playbook does not support running multiple SHA256 values, so I think we should keep the loop so we don't break anything.

Reply:
r u sure ? did you try to run it with an array ?